### PR TITLE
Improve logging and error handling for ecs actor.

### PR DESCRIPTION
**Pull request by swaggy**
@mikhail @diranged

### CODE REVIEW

**Comment by mikhail** [comment]
lgtm

File: kingpin/actors/aws/ecs.py [hunk 1]
```diff
@@ -363,25 +363,35 @@ def _tasks_done(self, tasks):
...
+                exit_code = container.get('exitCode', None)
+                if exit_code is None:
+                    self.log.error('Task {0} stopped without executing'.format(
+                        container['taskArn']))
+                    raise exceptions.RecoverableActorFailure()
+                if exit_code != 0:
+                    self.log.error('Task {0} finished with exit code {1}'.format(
+                        container['taskArn'], exit_code))
+                    raise exceptions.RecoverableActorFailure()
+                self.log.info('Task {0} finished with exit code 0'.format(
```
Comment (diranged) [inline kingpin/actors/aws/ecs.py]: I'd rather see `Task {0} finished successfully!`

File: kingpin/actors/aws/ecs.py [hunk 2]
```diff
@@ -363,25 +363,35 @@ def _tasks_done(self, tasks):
...
-                self.log.error('Task {0} exited with code {1}'.format(
-                    container['taskArn'], exit_code))
-                raise exceptions.RecoverableActorFailure()
+                exit_code = container.get('exitCode', None)
+                if exit_code is None:
+                    self.log.error('Task {0} stopped without executing'.format(
+                        container['taskArn']))
+                    raise exceptions.RecoverableActorFailure()
+                if exit_code != 0:
+                    self.log.error('Task {0} finished with exit code {1}'.format(
```
Comment (diranged) [inline kingpin/actors/aws/ecs.py]: I'd rather see: `Task {0} errored out with exit code: {1}`

**Comment by diranged** [comment]
Can you please build up some ECS integration tests?

**Comment by swaggy** [comment]
Agree that that's important. That's on the todo-list and I'm hoping to get to it after the service actor.

**Pull request merged**